### PR TITLE
Ltp networking fixes

### PR DESCRIPTION
**Pull request by pevik**
LTP networking fixes in net.tcp_cmds.

### CODE REVIEW

**Comment by pevik** [comment]
@richiejp FYI

File: tests/kernel/run_ltp.pm [hunk 1]
```diff
@@ -277,6 +278,12 @@ sub run {
         my $fin_msg    = "### TEST $test->{name} COMPLETE >>> ";
         my $cmd_text   = qq($test->{command}; echo "$fin_msg\$?");
         my $start_time = thetime();
+        my $set_rhost  = $is_network && $test->{command} =~ m/^finger01|ftp01|rcp01|rdist01|rsh01/;
+
+        if ($set_rhost) {
+            assert_script_run(q(export RHOST='localhost'));
```
Comment (richiejp) [inline tests/kernel/run_ltp.pm]: I hope we don't need to do many more workarounds like this.

**Comment by richiejp** [comment]
LGTM

File: tests/kernel/run_ltp.pm (hunk 1 above)
Comment (pevik) [inline tests/kernel/run_ltp.pm]: I hope as well. This is caused by combination of mrsh as rsh server and old tests requiring rsh. Best would be if LTP have converted all networking tests into new API.
I might find a way in the future to get rid of this.

File: tests/kernel/run_ltp.pm [hunk 2]
```diff
@@ -266,6 +266,7 @@ sub run {
         );
 
         script_run('ps axf');
+        script_run('netstat -ap');
```
Comment (richiejp) [inline tests/kernel/run_ltp.pm]: Apparently this is deprecated https://lwn.net/Articles/710533/#Comments

On my system `ss -ap` executes a lot faster although the output may be different.

Comment (pevik) [inline tests/kernel/run_ltp.pm]: You're right (I've read that article as well), I'll correct that. But quite a lot of LTP networking scripts require commands from net-tools. 8 use netstat, and there is even one whole [netstat test](https://github.com/linux-test-project/ltp/blob/master/testcases/network/tcp_cmds/netstat/netstat01) :-(.

Comment (pevik) [inline tests/kernel/run_ltp.pm]: #2288

**Pull request merged**